notebooks/tr-TR/fine_tuning_code_llm_on_single_gpu.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
<blockquote>Kod tamamlamasını sağlamak için tek yapmamız gereken, <code>get_code_complete</code> fonksiyonunu çağırmak ve tamamlanmasını istediğimiz ilk birkaç satırı önek olarak, boş bir dizeyi ise sonek olarak geçmek.</blockquote><p> </p><p> </p><p>Kodun tamamlamasını sağlamak için tek yapmamız gereken, <code>get_code_complete</code> fonksiyonunu çağırmak ve tamamlanmasını istediğimiz ilk birkaç satırı önek olarak, boş bir dizeyi ise sonek olarak geçmek.</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>sonek suffix mi 😮 ilginç 😂</p>